apps/condo/domains/billing/schema/BillingAccount.js

```diff
@@ -57,15 +54,15 @@ const BillingAccount = new GQLListSchema('BillingAccount', {
             type: Json,
             isRequired: true,
             hooks: {
-                validateInput: ({ resolvedData, fieldPath, addFieldValidationError }) => {
+                validateInput: (args) => {
```

Comment:
you can use `hasValidJsonStructure`

Reply:
I would rather do it in the next task, all along with new models